README.md

```diff
@@ -1,12 +1,30 @@
 # Docker-based Jenkins quick start examples
 
-This is a experiment repository for GSoC 2023 docker based jenkins quick start examples project 
+* This is an experiment repository for the GSoC 2023 docker-based Jenkins quick start examples project 
 
 ## Proposal
 
-You can find the proposal for this project [here](https://docs.google.com/document/d/1ZpPihadYqpAvR20rxZkTD2SVpf34E6YMzg6opU6yHAg/edit#heading=h.lntg56ljm653).
+* You can find the proposal for this project [here](https://docs.google.com/document/d/1ZpPihadYqpAvR20rxZkTD2SVpf34E6YMzg6opU6yHAg/edit#heading=h.lntg56ljm653).
 
 ## Links
 
 * Meeting notes for this project can be found [here](https://docs.google.com/document/d/1yij9OvM2_92My3vqjn6u8ABHjXcyy0a7O6oM30b6ctM/edit).
 * Project page can be found [here](https://www.jenkins.io/projects/gsoc/2023/projects/docker-compose-build/).
+
+## How to Run with Gitpod  
+* Gitpod is a cloud development environment for teams to efficiently and securely develop software.
+* Gitpod supports many IDEs, such as VScode, IntelliJ, and many more.  
+
+ ### How to setup the repo in Gitpod?
+* For starting your gitpod workspace you can use `gitpod.io/#` as a prefix on any GitHub, gitlab, or bitbucket repo URL
+* For our repo you can get the gitpod workspace from [here](https://gitpod.io/#https://github.com/ash-sxn/GSoC-2023-docker-based-quickstart)  
+* But If you do plan to use gitpod in the future, it's recommended to install the gitpod extension which will create a launch with gitpod button on every GitHub repo you visit, It can be found [here](https://chrome.google.com/webstore/detail/gitpod-online-ide/dodmmooeoklaejobgleioelladacbeki) for Chromium and [here](https://addons.mozilla.org/firefox/addon/gitpod/) for Firefox.
```

Comment:
```suggestion
* But If you do plan to use gitpod in the future, it's recommended to install the gitpod extension which will create a launch with the gitpod button on every GitHub repo you visit.
It can be found [here](https://chrome.google.com/webstore/detail/gitpod-online-ide/dodmmooeoklaejobgleioelladacbeki) for Chromium and [here](https://addons.mozilla.org/firefox/addon/gitpod/) for Firefox.
```